### ISSUE TITLE
Not reading logs from S3 location

### ISSUE THREAD

**Issue by akashsark** [AppsFlyer/elb-log-replay#7]
When we are trying to put the logs path of S3 rather than local , the code is not working

**Comment by eranharel** [comment]
I don't think there's any support for log files from S3 here.
Are you requesting a feature?

**Comment by akashsark** [comment]
Yeah if we don't have any such support can we add it ? Basically the intent is to read logs from S3 buckets and since the logs size will be huge so we usually keep them as .gz, so if we have that support as well for this then it will help us in long run.

**Comment by eranharel** [comment]
Do you want to send a PR?
